### PR TITLE
fix: image preview display error when image.alt includes '<' or '>'

### PR DESCRIPTION
**Pull request by PENEKhun**
### Describe the bug
When specifying a preview image in a post,  
if you specify a value of < or > for `alt`, the image will not display properly because it encroaches on the HTML tag range.

### To Reproduce

1. Create a new post.
2. In the post properties, write image.alt to contain < or >, as follows

- example
```yml
image:
  path: /commons/devices-mockup.png
  alt: get device image -> show mockup
```

### Expected behavior && Logs/Screenshots

- At Home (root path)
image  

- At Post
image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (refactoring and improving code)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How has this been tested

- [X] I have run `bash ./tools/test` (at the root of the project) locally and passed
- [X] I have tested this feature in the browser

### CODE REVIEW

**Review by cotes2020 — CHANGES_REQUESTED** [review]
It would be better to use Jekyll's `xml_escape` filter.

File: _layouts/home.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
             {% assign src = post.img_path | append: '/' | append: src | replace: '//', '/' %}
           {% endunless %}
 
-          {% assign alt = post.image.alt | default: 'Preview Image' %}
+          {% assign alt = post.image.alt | default: 'Preview Image' | replace: '<', '&lt;' | replace: '>', '&gt;' %}
```
Comment (cotes2020) [inline _layouts/home.html:57]:
```suggestion
         {% assign alt = post.image.alt | xml_escape | default: 'Preview Image' %}
```

File: _layouts/post.html [hunk 2]
```diff
@@ -31,7 +31,7 @@ <h1 data-toc-skip>{{ page.title }}</h1>
   {% if page.image %}
     {% capture src %}src="{{ page.image.path | default: page.image }}"{% endcapture %}
     {% capture class %}class="preview-img{% if page.image.no_bg %}{{ ' no-bg' }}{% endif %}"{% endcapture %}
-    {% capture alt %}alt="{{ page.image.alt | default: "Preview Image" }}"{% endcapture %}
+    {% capture alt %}alt="{{ page.image.alt | default: "Preview Image" | replace: "<", "&lt;" | replace: ">", "&gt;" }}"{% endcapture %}
```
Comment (cotes2020) [inline _layouts/post.html:34]:
```suggestion
    {% capture alt %}alt="{{ page.image.alt | xml_escape | default: "Preview Image" }}"{% endcapture %}
```

**Review by cotes2020 — CHANGES_REQUESTED** [review]

**Comment by PENEKhun** [comment]
thanks you for suggestion.

File: _layouts/home.html [hunk 3]
```diff
@@ -54,7 +54,7 @@
             {% assign src = post.img_path | append: '/' | append: src | replace: '//', '/' %}
           {% endunless %}
 
-          {% assign alt = post.image.alt | default: 'Preview Image' | replace: '<', '&lt;' | replace: '>', '&gt;' %}
+         {% assign alt = post.image.alt | xml_escape | default: 'Preview Image' %}
```
Comment (cotes2020) [inline _layouts/home.html:57]: Be careful with indentation:

```suggestion
          {% assign alt = post.image.alt | xml_escape | default: 'Preview Image' %}
```

**Review by cotes2020 — CHANGES_REQUESTED** [review]

**Review by cotes2020 — APPROVED** [review]

**Pull request merged**